.teamcity/settings.kts

```diff
@@ -262,11 +262,7 @@ object RunAllUnitTests : BuildType({
 				set -o nounset
 				set -o pipefail
 
-				export HOME="/calypso"
 				export NODE_ENV="test"
-				export CHROMEDRIVER_SKIP_DOWNLOAD=true
-				export PUPPETEER_SKIP_DOWNLOAD=true
```

Comment:
Are you sure these env variables are really unused? They are checked by `puppeteer` and `chromedriver` install scripts to avoid downloading large binaries from cloud. Has anything happened that would make them obsolete?

Reply:
They are part of the Docker image already: https://github.com/Automattic/wp-calypso/blob/trunk/Dockerfile.base#L12

Reply:
Thanks, that makes sense :+1: